etcdserver/api/v3rpc/grpc.go

```diff
@@ -36,8 +36,11 @@ func Server(s *etcdserver.EtcdServer, tls *tls.Config) *grpc.Server {
 	}
 	opts = append(opts, grpc.UnaryInterceptor(newUnaryInterceptor(s)))
 	opts = append(opts, grpc.StreamInterceptor(newStreamInterceptor(s)))
-
+	// set MaxMsgSize be a bit higher than that of MaxRequestBytes bypasses grpc request size check
```

Comment:
ok, so should the server limit be `MaxRequestBytes + grpcOverheadBytes`  with `const grpcOverheadBytes = 512 * 1024` to be careful about the rpc overhead without totally abandoning the grpc request size checking?